app/src/main/res/values/strings.xml

```diff
@@ -82,6 +82,7 @@
   <string name="patiententry_consent_second_para">Anyone authorized by you, or the healthcare provider for whom you work, will have access to the patient data and information you enter here. By using the app you confirm that the patient understands who will have access to the data obtained through this app and consents to that access.</string>
   <string name="patiententry_consent_whatsapp_sms_reminders">WhatsApp / SMS Reminders</string>
   <string name="patiententry_invalid_age_error">Age cannot be more than 120 years</string>
+  <string name="patiententry_age_exceeds_min_limit_error">Age cannot be equal to 0 years</string>
```

Comment:
Did we get this text verified with someone from the design team?

Reply:
Yes, Varenya confirmed this.